skit.API/appsettings.Development.json

```diff
@@ -12,5 +12,13 @@
     "JwtKey": "",
     "JwtIssuer": "",
     "Expires": ""
+  },
+  "SMTP": {
+    "SmtpUrl": "",
+    "SmtpPort": 100,
+    "SmtpLogin": "",
+    "SmtpPassword": "",
+    "SmtpSenderMail": "",
+    "SmtpSenderName": ""
```

Comment:
Czekamy na jakiegoś maila? A jeżeli tak to czy takie dane powinny być tutaj trzymane?

Reply:
Na razie czekamy, dla testu każdy może jakąś swoją skrzynkę założyć